js/utils/synthutils.js

```diff
@@ -742,16 +741,15 @@ function Synth() {
 
     this.samplesQueue = []; // Samples that need to be loaded at start.
 
-    let that = this;
     require(SOUNDSAMPLESDEFINES, function() {
```

Comment:
without arrow function `that` !== `this`. Don't send a PR without testing. This one pops up on load itself.